compiler/src/main/java/com/squareup/anvil/compiler/codegen/BindingModuleGenerator.kt

```diff
@@ -190,25 +190,21 @@ internal class BindingModuleGenerator(
 
         val replacedBindings = allContributedClasses
           .flatMap { classReference ->
-            classReference.annotations.single { it.fqName == annotationFqName }.replaces()
+            classReference.annotations
+              .find(annotationName = annotationFqName, scope = scope)
+              .flatMap { it.replaces() }
           }
 
         return allContributedClasses
           .asSequence()
           .filterNot { it in replacedBindings }
           .filterNot { it in bindingsReplacedInDaggerModules }
           .filterNot { it in excludedNames }
-          .filter { clazz ->
+          .flatMap { clazz ->
```

Comment:
This and the change above is literally the only important change. That was simple 😅